quarkchain/cluster/filter.py

```diff
@@ -87,8 +87,15 @@ def _get_logs(self, blocks: List[MinorBlock]) -> List[Log]:
         """Given potential blocks, re-run tx to find exact matches."""
         ret = []
         for b_i, block in enumerate(blocks):
-            for i in range(len(block.tx_list or [])):
-                r = block.get_receipt(self.db.db, i)
+            deposit_hlist = (
```

Comment:
I would suggest adding one line comment to illustrate missing case (old version db)

Reply:
done. put the older version db handling into the `block. get_receipt`, by returning a fake tx receipt